src/ocamlorg_frontend/css/partials/bg_sections.css

```diff
@@ -2,6 +2,11 @@
 
 .section-mild-contrast {
     background: url("/img/innerbottombg.png") left bottom no-repeat, url("/img/innertopbg.png") right top no-repeat;
-    @apply bg-legacy-mild-contrast border-b border-t;
+    @apply bg-sand border-b border-t;
+  }
+
+.dark-section-mild-contrast {
+  background: linear-gradient(45deg, #010105, #10162D);
+    @apply border-none;
```

Comment:
We need to wrap the file content in `@layer utilities`, then the `dark:dark-section-milt-contrast` class will work as expected.